docs/modules/k3s.md

```diff
@@ -0,0 +1,79 @@
+# K3s
+
+Not available until the next release of testcontainers-go <a href="https://github.com/testcontainers/testcontainers-go"><span class="tc-version">:material-tag: main</span></a>
+
+## Introduction
+
+The Testcontainers module for K3s.
+
+## Adding this module to your project dependencies
+
+Please run the following command to add the K3s module to your Go dependencies:
+
+```
+go get github.com/testcontainers/testcontainers-go/modules/k3s
+```
+
+## Usage example
+
+<!--codeinclude-->
+[Creating a K3s container](../../modules/k3s/k3s.go)
```

Comment:
I'll narrow down the code snippet to the only block calling the RunContainer function

Reply:
Ohh, perfect only, done that now, thanks a lot!